src/main/asciidoc/_chapters/external_apis.adoc

```diff
@@ -341,7 +376,8 @@ curl -vi -X DELETE \
 
 |===
 
-.Endpoints for `Scan` Operations
+[[stateful.scanner.endpoints]]
+.Endpoints of Stateful Scanner
```

Comment:
I renamed this already documented "Scan operations endpoints" here to stateful scanner so that it is easier to distinguish it from the stateless scanner.
What so you think about this?

Reply:
A better name would be "Stateful endpoints for scan operations", as it's the endpoint that is stateful/stateless, not the scanner operation.

Reply:
The same goes for the title of the stateless endpoints.

Reply:
Thanks for the feedback, renamed them.